examples/storage_system.go

```diff
@@ -20,7 +20,7 @@ func main() {
 		os.Getenv("ONEVIEW_OV_DOMAIN"),
 		os.Getenv("ONEVIEW_OV_ENDPOINT"),
 		false,
-		2000,
+		2200,
 		"*")
 
 	// Create storage system
```

Comment:
Remove username and password

Reply:
you mean the for below line?
storageSystem := ov.StorageSystem{Hostname: "172.18.30.1", Username: "dcs", Password: "dcs", Family: "StoreVirtual", Description: "<description>"}

Reply:
yes, also use variable here also

Reply:
done

Reply:
okay, have updated.